sdk/cpp/core/src/ydk.cpp

```diff
@@ -117,21 +117,23 @@ static ydk::EncodingFormat get_real_encoding(EncodingFormat encoding)
 {
     switch(encoding)
     {
-        case XML:
-            return ydk::EncodingFormat::XML;
         case JSON:
             return ydk::EncodingFormat::JSON;
+        case XML:
+        default:
+            return ydk::EncodingFormat::XML;
     }
 }
 
 static ydk::Protocol get_real_protocol(Protocol protocol)
 {
     switch(protocol)
     {
-        case Netconf:
-            return ydk::Protocol::netconf;
         case Restconf:
             return ydk::Protocol::restconf;
+        case Netconf:
+        default:
```

Comment:
Same as above. Better to avoid default switch cases

Reply:
The compiler logged warning message pointing that 'default' case is missing. I was taught always include 'default' case. If not included the 'switch' must be re-factored to if-else statements.